packages/ui/src/components/DateInput/__tests__/__snapshots__/index.test.tsx.snap

```diff
@@ -4512,7 +4512,7 @@ exports[`DateInput renders correctly with date-fns locale ru 1`] = `
                   <p
                     class="e17s8tvr0 cache-1re7cvs-StyledText-StyledText e1tg3t120"
                   >
-                    январь 2021 г.
```

Comment:
This should be reverted

Reply:
update to node 18.13

Reply:
grrrr form tests make me mad @matthprost